### DISCUSSION TITLE
Redshift (bluelight filter) widget

### DISCUSSION THREAD

**Discussion by nullbyto** [qtile/qtile#3727] [Ideas] [answered]
Hello,

We need a blue light filter (redshift for example) widget indicator for the bar.
With on/of switch and scrolling and percentage.
It's missing I'm surprised.

**Comment by elParaguayo** [comment, marked as answer]
Why are you surprised? As far as I'm aware, you're the first person to ask for it.

I can add to the "to do" list, but it's a long list so you may want to have a go at making it yourself (this should help: http://docs.qtile.org/en/stable/manual/howto/widget.html)

**Reply by nullbyto** [reply]
It's a common widget for bars.
I know there are other more important stuff to do in this list :D.

In the mean time i will use a [script ](https://github.com/VineshReddy/polybar-redshift) or maybe when i have time i'll make a widget.
